content/spin/install.md

```diff
@@ -152,6 +155,26 @@ $ spin --help
 $ rustup update
 ```
 
+## Installing Default Templates and Plugins
```

Comment:
It might be good to call out what the default template and plug-ins are. Whether in this doc, or another place, not sure.

At least the repos we consider default (spin, spin-python-sdk, spin-js-sdk, and the two plugins.)

Reply:
Yeah i was wondering if we want to textually list them or whether the installation code is descriptive enough. We should probably define what is default. in my head its "language utilities" and cloud plugin.

Reply:
This appears to be resolve in the source.